lib/cc/cli/init.rb

```diff
@@ -9,6 +9,7 @@ class Init < Command
 
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+          create_default_configs
           fatal "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
```

Comment:
It seems odd to have a fatal output if we successfully created default configs. Maybe the message should mention that we generated them?